app/utils/cli-install.js

```diff
@@ -0,0 +1,86 @@
+const pify = require('pify');
+const fs = require('fs');
+const path = require('path');
+const Registry = require('winreg');
+
+const {cliScriptPath} = require('../config/paths');
+
+const lstat = pify(fs.lstat);
+const readlink = pify(fs.readlink);
+const unlink = pify(fs.unlink);
+const symlink = pify(fs.symlink);
+
+const target = '/usr/local/bin/hyper';
+const source = cliScriptPath;
+
+const checkInstall = () => {
+  return lstat(target)
+    .then(stat => stat.isSymbolicLink())
+    .then(() => readlink(target))
+    .then(link => link === source)
+    .catch(err => {
+      if (err.code === 'ENOENT') {
+        return false;
+      }
+      throw err;
+    });
+};
+
+const createSymlink = () => {
+  return unlink(target)
+    .catch(err => {
+      if (err.code === 'ENOENT') {
+        return;
+      }
+      throw err;
+    })
+    .then(() => symlink(source, target));
+};
+
+exports.addSymlink = () => {
+  return checkInstall().then(isInstalled => {
+    if (isInstalled) {
+      return Promise.resolve();
+    }
+    return createSymlink();
+  });
+};
+
+exports.addBinToUserPath = () => {
+  // Can't use pify because of param order of Registry.values callback
+  return new Promise((resolve, reject) => {
+    const envKey = new Registry({hive: 'HKCU', key: '\\Environment'});
+    envKey.values((err, items) => {
+      if (err) {
+        reject(err);
+        return;
+      }
+      // C:\Users\<user>\AppData\Local\hyper\app-2.0.4\resources\bin
```

Comment:
`// C:\Users\<user>\AppData\Local\hyper\app-<version>\resources\bin`

Reply:
I'll fix this

Reply:
fixed